crates/ruff/src/cst/matchers.rs

```diff
@@ -54,6 +54,16 @@ pub fn match_import_from<'a, 'b>(module: &'a mut Module<'b>) -> Result<&'a mut I
     }
 }
 
+pub fn match_aliases<'a, 'b>(
+    import_from: &'a mut ImportFrom<'b>,
+) -> Result<&'a mut Vec<ImportAlias<'b>>> {
+    if let ImportNames::Aliases(aliases) = &mut import_from.names {
+        Ok(aliases)
+    } else {
+        bail!("Expected ImportNames::Aliases")
+    }
```

Comment:
What's the reason for using `bail` here instead of returning `Err` and propagating the error (we'll have to replace all `bail`, `warn_once` at some point in the future if we want to build a python API)

Reply:
(fwiw there's anyhow integration with pyo3 that translates them into `RuntimeError`, but i agree that proper error types are better)

Reply:
Oh, I think I confused what `bail!` does. Does bail return an `Err`? If so, then I think that's fine for now.